userguide/tutorials/faq.adoc

```diff
@@ -64,3 +64,9 @@ Data is anonymous and not shared with any third party, and no financial data is
 To disable tracking, specify `org.killbill.server.updateCheck.skip=true` in your killbill.properties.
 
 We are relying on this data to help us focus our development efforts. If your environment prevents you from sharing this automatically, but would like to help, please reach out to the https://groups.google.com/g/killbilling-users[Community] (mailing list) instead and describe your deployment. Thanks in advance!
+
+== How do I troubleshoot Kill Bill?
+
+See our https://docs.killbill.io/latest/java_client.html#_troubleshooting[debugging documentation].
```

Comment:
It's a bit odd to point to the Java client specifically. I think @reshmabidikar has written more generic instructions.

Worth maybe having a separate troubleshooting page as we re-organize the docs?

Reply:
Yes, agreed, the Java client doc is more to do with the Java client and not troubleshooting Kill Bill in general. We do have FAQ sections in some docs like [Getting Started](https://docs.killbill.io/latest/getting_started.html), [Development Doc](https://docs.killbill.io/latest/development.html), etc. But, yes, we can think of adding a separate Troubleshooting page.